experimental/linting_tool/lib/widgets/lint_expansion_tile.dart

```diff
@@ -268,14 +274,19 @@ class _NewProfileDialog extends StatelessWidget {
   }
 }
 
-class _ExistingProfileDialog extends StatelessWidget {
-  const _ExistingProfileDialog({
+class ExistingProfileDialog extends StatefulWidget {
+  const ExistingProfileDialog({
     Key? key,
     required this.rule,
   }) : super(key: key);
 
   final Rule rule;
 
+  @override
+  State<ExistingProfileDialog> createState() => ExistingProfileDialogState();
+}
+
+class ExistingProfileDialogState extends State<ExistingProfileDialog> {
```

Comment:
This one probably needs hiding. I think.

Reply:
whoops. fixt.